src/app/app.routes.js

```diff
@@ -14,15 +14,118 @@ angular.module('wisboo').config([
           }
         }
       })
-      .state('home', {
-        url: '/',
+      .state('books', {
+        url: '/books',
         views: {
           main: {
-            templateUrl: '../app/components/home/home.html'
+            templateUrl: '../app/components/books/index.html'
+          }
+        }
+      })
+      .state('new_book', {
+        url: '/books/new',
+        views: {
+          main: {
+            templateUrl: '../app/components/books/new.html'
+          }
+        }
+      })
+      .state('book', {
+        url: '/books/:id',
+        views: {
+          main: {
+            templateUrl: '../app/components/books/show.html',
+            controller: 'idController'
+          },
+          'comments@book': {
+            templateUrl: '../app/components/comments/index.html'
+          },
+          'comments_new@book': {
+            templateUrl: '../app/components/comments/new.html'
+          }
+        }
+      })
+      .state('edit_book', {
+        url: '/books/:id/edit',
+        views: {
+          main: {
+            templateUrl: '../app/components/books/edit.html'
+          }
+        }
+      })
+      .state('edit_comment', {
+        url: '/comments/:id/edit',
+        views: {
+          main: {
+            templateUrl: '../app/components/comments/edit.html'
+          }
+        }
+      })
+      .state('signup', {
+        url: '/signup',
+        views: {
+          main: {
+            templateUrl: '../app/components/users/registration.html'
+          }
+        }
+      })
+      .state('login', {
+        url: '/login',
+        views: {
+          main: {
+            templateUrl: '../app/components/users/sessions_new.html'
+          }
+        }
+      })
+      .state('users', {
+        url: '/users',
+        views: {
+          main: {
+            templateUrl: '../app/components/users/index.html'
+          }
+        }
+      })
+      .state('user', {
+        url: '/users/:id',
+        views: {
+          main: {
+            templateUrl: '../app/components/users/show.html',
+            controller: 'idController'
+          }
+        }
+      })
+      .state('user.following', {
+        url: '/following',
+        templateUrl: '../app/components/users/following.html'
+      })
+      .state('user.followers', {
+        url: '/followers',
+        templateUrl: '../app/components/users/followers.html'
+      })
+      .state('user.favorite_books', {
+        url: '/favorite_books',
+        templateUrl: '../app/components/users/favorite_books.html'
+      })
+      .state('user.wish_list', {
+        url: '/wish_list',
+        templateUrl: '../app/components/users/wish_list.html'
+      })
+      .state('edit_user', {
+        url: '/users/:id/edit',
```

Comment:
the profile to edit should always be the one of the user that it's currently logged in, no need to have the id